lib/src/base-styles/_nx-lists.scss

```diff
@@ -10,8 +10,9 @@
 .nx-list {
   @include container-vertical;
   margin: $nx-spacing-sm 0 $nx-spacing-md 0;
+  padding: 0;
 
-  &, > ul, > ol {
+  ul, ol {
```

Comment:
Why this change?

Reply:
> Why this change?

This is what caused the problem reported by Andrey in IQ. In this case `<ul className="nx-list">` loses the margin that are applied to `.nx-list`. This is incorrect.

Reply:
> In this case <ul className="nx-list"> loses the margin that are applied to .nx-list. This is incorrect.

In what valid scenario would a `<ul classname="nx-list">` be an immediate child of a `<div className="nx-list">`?

Reply:
> > In this case  loses the margin that are applied to .nx-list. This is incorrect.
> 
> In what valid scenario would a `<ul classname="nx-list">` be an immediate child of a `<div className="nx-list">`?

Adjusted here: https://github.com/sonatype/sonatype-react-shared-components/pull/18/commits/08dfe7f04fdd3ffabf59b3d45d6adab914c0dc84